modules/core/src/main/java/org/apache/fluo/core/oracle/OracleServer.java

```diff
@@ -161,11 +161,20 @@ public void run() {
           }
         }
       };
+      TimerTask logTask = new TimerTask() {
+        @Override
+        public void run() {
+          log.warn("Loging the current timestamp: ", currentTs);
```

Comment:
Don't really need `logging` message.  Also it may be better to log this at INFO, as WARN implies there may be a problem.  Could do something like the following.

```java
   log.info("Current timestamp: ", currentTs);
```